### ISSUE TITLE
[TW-985] Tasks imported from .txt behave strangely

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1020]
_Elizabeth Maxson on 2010-10-03T15:15:45Z says:_ 

 I imported several tasks from a .txt file formatted with one task per line.  Taskwarrior recognized these and imported them, but they behaved strangely after import: actions applied to one of them would affect all of them, which was undesirable.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2010-10-03T15:15:45Z
Modified: 2014-02-09T02:13:25Z
```

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2010-10-03T15:26:35Z says:_ 

 Thank you for the bug report.  This sounds serious.  Can you please provide:
*  OS and version
*  Task version (use the "task version" command)
*  Did you build from source, or install a package?

Please take a look at your pending.data file and tell me if the uuid:"..." values are all the same?  That is what I suspect.

**Comment by taskwarrior** [comment]
_Elizabeth Maxson on 2010-10-04T01:40:28Z says:_ 

 Version is 1.9.1 built for linux ncurses.  It was installed from the .deb from the ultrafredde ppa.

I'm running Ubuntu 10.04 

The tasks added from a text file share a uuid in that they share the quality of not having one, and they have the same ten digit entry number.

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2010-10-05T01:12:23Z says:_ 

 The good news is that this bug was fixed in this 1.9.3 commit, a month ago:

    * commit 1cc67e98957bccf1f7b2277076def8c80c0d9341
    | Author: Paul Beckingham <paul(at)beckingham.net>
    | Date:   Thu Sep 2 21:32:33 2010 -0400

But the bad news is that there is no workaround in 1.9.1 or 1.9.2, and that 1.9.3 isn't going to be available for a couple more weeks, when the beta wraps up.  I recommend waiting for the 1.9.3 release, unless of course you want to use the color themes introduced in 1.9.2.

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2010-10-05T01:15:15Z says:_ 

 Bug reporter named/thanked in the AUTHORS file.
